l3build-check.lua

```diff
@@ -34,7 +34,7 @@ local luatex_version   = status.luatex_version
 
 local len              = string.len
 local char             = string.char
-local format           = string.format
+local frmt             = string.format
```

Comment:
The name `format` is reserved for TeX format name.
The function is named after the consonants only (somehow like `rnd` for math.random.)